basis/windows/errors/errors.factor

```diff
@@ -758,11 +755,7 @@ CONSTANT: expected-io-errors
     expected-io-errors member? ;
 
 : expected-io-error ( error-code -- )
```

Comment:
this is weird that it drops the error code and throws ``GetLastError``.... is that the correct?